flexget/plugins/input/from_piratebay.py

```diff
@@ -145,13 +150,14 @@ def on_task_input(self, task, config):
             else:
                 list_url = f'{url}/precompiled/data_top100_48h.json'
         elif category:
-            list_url = f'{url}/q.php?q=category:{category}'
+            list_url = f'{url}/q.php'
```

Comment:
This could be done in terms of `query`, but I didn't want to change the precedence of anything here, so this seemed like the cleanest option for now.